queries.sql

```diff
@@ -31,4 +31,57 @@ SELECT full_name as owner_full_name, name as animal_name FROM OWNERS
 SELECT full_name, COUNT(*) as total_animals FROM OWNERS 
   JOIN ANIMALS ON owners.id = animals.owner_id 
   GROUP BY FULL_NAME
-  ORDER BY total_animals DESC;
\ No newline at end of file
+  ORDER BY total_animals DESC;
+
+SELECT vets.name, visits.date_visited FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  WHERE vets.name = 'William Tatcher'
+  ORDER BY visits.date_visited DESC;
```

Comment:
- [ ] Good job working on this query but this returns all animals seen by the vet. Please limit it to get the last animal.